src/app/modules/ix-table2/base-data-provider.ts

```diff
@@ -61,6 +62,11 @@ export class BaseDataProvider<T> implements DataProvider<T> {
     this.updateCurrentPage(this.allRows);
   }
 
+  setExpandedRow(row: T): void {
+    this.expandedRow = row;
+    this.expandedRow$.next(row);
```

Comment:
Does not appear to be used.

Reply:
Ah, there is another implementation based on EventEmitter that was added recently.